### ISSUE TITLE
Fabric 1.19 crash on startup, both with other mods and alone

### ISSUE THREAD

**Issue by mhom291** [malte0811/FerriteCore#83]
Launching the game with FerriteCore crashes on startup, both with and without other mods.

Here's latest.log
[latest.log](https://github.com/malte0811/FerriteCore/files/9200473/latest.log)

**Comment by malte0811** [comment]
Does `C:\Users\censor\AppData\Roaming\PolyMC\instances\1.19 Fabric\.minecraft\config` exist, and if it does, is it a file or a folder?

**Comment by mhom291** [comment]
it is a folder

**Comment by malte0811** [comment]
Hm, interesting. That call is not supposed to fail for existing folders AFAIK, but I can wrap it in a check anyway.

**Comment by puppie1997** [comment]
Has this been fixed? I also have this issue on 1.19.2. I haven't updated to 1.19.3 because I'm waiting on other mods

**Comment by malte0811** [comment]
I've added a workaround, but I still don't entirely understand how this is possible. Is there anything strange about your config directory (or your MC directory)? Network folder, symlinks, strange symbols in the path, anything like that?

**Comment by HB-Stratos** [comment]
> @malte0811  I've added a workaround, but I still don't entirely understand how this is possible. Is there anything strange about your config directory (or your MC directory)? Network folder, symlinks, strange symbols in the path, anything like that?


I'm facing a very similar crash. I am using symlinks. 
What I have done is that I have two different minecraft servers proxied together using velocity.
To keep them in sync I have extracted the ``mods`` and ``config`` folder from their respective server folders into a higher folder. The symlink to ``mods`` works perfectly fine, all mods load correctly. The ``config`` symlink also worked for reading and writing in ``config`` for some installed mods, but when I copied my whole modpack including ferritecore over, it fails to launch when the symlink is set, and launches correctly with the symlink not set. (it may be important to note that I am using relative symlinks, aka ``ln -s ../config``)
Interestingly when manually copying all contents from normal ``config`` into the symlinked ``config``, then restoring the symlink and launching the server, it launches correctly.
That would imply to me that it is specifically an issue with how ferritecore writes to a symlinked folder. 

[latest.log](https://github.com/malte0811/FerriteCore/files/13537458/latest.log)
